unity/EditorPlugin/UnitTesting/UnityEditorTestLauncher.cs

```diff
@@ -10,21 +12,24 @@
 using UnityEngine.Events;
 using TestResult = JetBrains.Platform.Unity.EditorPluginModel.TestResult;
 
-
 namespace JetBrains.Rider.Unity.Editor.UnitTesting
 {
   public class UnityEditorTestLauncher
   {
     private readonly UnitTestLaunch myLaunch;
+    private readonly Lifetime myLifetime;
     private const string RunnerAddlistener = "AddListener";
     private const string LauncherRun = "Run";
+    private const string StopRun = "StopRun";
     private const string MTeststartedevent = "m_TestStartedEvent";
     private const string MRunfinishedevent = "m_RunFinishedEvent";
     private const string MTestfinishedevent = "m_TestFinishedEvent";
     private const string MEditmoderunner = "m_EditModeRunner";
+    private const string MRunner = "m_Runner";
     private const string TestNames = "testNames";
     private const string TestNames56 = "names";
     private const string EditModeLauncher = "UnityEditor.TestTools.TestRunner.EditModeLauncher";
+    private const string PlayModeLauncher = "UnityEditor.TestTools.TestRunner.PlaymodeLauncher";
     private const string TestRunnerFilter = "UnityEngine.TestTools.TestRunner.GUI.TestRunnerFilter";
     private const string TestPlatform = "UnityEngine.TestTools.TestPlatform";
 
```

Comment:
I'd consider inlining these constants if they're only used in one place. Otherwise, you only have to look them up when reading the code.